backend/tracim_backend/views/core_api/workspace_controller.py

```diff
@@ -790,6 +797,48 @@ def unarchive_content(self, context, request: TracimRequest, hapic_data=None) ->
         with new_revision(session=request.dbsession, tm=transaction.manager, content=content):
             api.unarchive(content)
         return
+    
+
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__CONTENT_ENDPOINTS])
+    @check_right(is_contributor)
+    @hapic.input_path(WorkspaceAndContentIdPathSchema())
+    @hapic.input_body(SetContentMarkedAsTemplateSchema())
+    @hapic.output_body(NoContentSchema(), default_http_code=HTTPStatus.NO_CONTENT)
+    def mark_content_as_template(self, context, request: TracimRequest, hapic_data=None) -> None:
```

Comment:
Shouldn't the function name be `set_content_template`? As the function can either set is_template to True or to False, not only changing it to True.